pylibczi/CziFile.py

```diff
@@ -17,12 +17,11 @@
 # Parent class for python wrapper to libczi file for accessing Zeiss czi image and metadata.
 
 import io
-import time
+import numpy as np
+from lxml import etree as etree
```

Comment:
I don't think this `as etree` is needed. Simply `from lxml import etree` should be enough no?